README.adoc

```diff
@@ -69,6 +69,8 @@ NOTE: If your IDE has the Spring Initializr integration, you can complete this p
 
 NOTE: You can also fork the project from Github and open it in your IDE or other editor.
 
+NOTE: For Spring 3.0 you need Java 17 or later, regardless using Spring Initializr or not.
```

Comment:
Our usual fix for this is to remove the Java version from the Actuator link rather than add a note. Then the default Java version in the Initializr is correct.

Reply:
Now that I look more closely at the wording, I see that it should be:

NOTE: For Spring 3.0 you need Java 17 or later, regardless of whether you use Spring Initializr.

Sorry I didn't catch it the first time. I was focused on the Initializr link.